systest/backup/common/utils.go

```diff
@@ -91,6 +105,257 @@ func CopyToLocalFs(t *testing.T) {
 	require.NoError(t, testutil.DockerCp(srcPath, copyBackupDir))
 }
 
+//***************************************New Adds
+
+func RemoveContentsOfPerticularDir(t *testing.T, dir string) {
+	d, err := os.Open(dir)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer d.Close()
+	names, err := d.Readdirnames(-1)
+	if err != nil {
+		t.Fatal(err)
+	}
+	for _, name := range names {
+		if name != ".gitkeep" {
+			err = os.RemoveAll(filepath.Join(dir, name))
+			if err != nil {
+				t.Fatal(err)
+			}
+		}
+	}
+}
+
+func AddNamespaces(t *testing.T, namespaceQuant int, header http.Header, customAdminURL string) uint64 {
```

Comment:
can we consider modifying the `ommon.CreateNamespace` function itself to allow you to do this?

Reply:
Resolved,

Updated `common.Createnamespace()`. Passing the struct which includes 2 additional parameters